h2o-py/h2o/explanation/_explain.py

```diff
@@ -1291,6 +1294,17 @@ def ice_plot(
     if frame.type(column) == "string":
         raise ValueError("String columns are not supported!")
 
+    if model.parms['distribution']['actual_value'] != 'bernoulli' and (binary_score_form == "logodds" or binary_score_form == "probability"):
+        raise ValueError("binary_score_form option is only supported for binomial models!")
+
+    if model.parms['distribution']['actual_value'] == 'bernoulli' and binary_score_form is None:
+        binary_score_form = "probability"
+
+    if binary_score_form != "logodds" and binary_score_form != "probability" and binary_score_form is not None:
```

Comment:
nit/optional: Maybe a little more ellegant would be `if binary_score_form not in ["logodds", "probability", None]:`

Reply:
f3437b8